site/src/@ndlib/gatsby-theme-marble/components/Layout/PageWrapper/Footer/index.js

```diff
@@ -14,66 +14,62 @@ export const Footer = () => {
-            <p className='contact-info'>
-              <div className='address' property='address' typeof='PostalAddress'>
-                <span property='streetAddress'>100 Moose Krause Circle</span><br />
-                <span property='addressLocality'>Notre Dame</span>, <span property='addressRegion'>IN</span> <span property='postalCode'>46556</span> <span property='addressCountry'>USA</span>
-              </div>
-              <div className='footer-phone' property='telephone' content='+1 574-631-5466'>Phone <a href='tel:574-631-5466'>(574) 631–5466</a></div>
-              <div className='footer-email' property='email'><a rel='noopener' href='mailto:sniteart@nd.edu'>sniteart@nd.edu</a></div>
-            </p>
-            <a
-              href='https://library.nd.edu'
-              sx={sx.imageWrapper}
-            >
-              <img
-                src={libraryLogo}
-                sx={sx.image}
-                alt='Hesburgh Library'
-              />
-            </a>
-            <p className='contact-info'>
-              <div className='address' property='address' typeof='PostalAddress'>
-                <span property='streetAddress'>284 Hesburgh Library</span><br />
-                <span property='addressLocality'>Notre Dame</span>, <span property='addressRegion'>IN</span> <span property='postalCode'>46556</span> <span property='addressCountry'>USA</span>
-              </div>
-              <div className='footer-phone' property='telephone' content='+1 574-631-6679'>Phone <a href='tel:574-631-6679'>(574) 631–6679</a></div>
-              <div className='footer-email' property='email'><a rel='noopener' href='mailto:asklib@nd.edu'>asklib@nd.edu</a></div>
-            </p>
-          </div>
-          <div sx={sx.copyright} className='copyright'><a href='https://www.nd.edu/copyright/'>&copy; 2020</a> <a href='https://www.nd.edu'>University of Notre Dame</a></div>
-        </div>
-      </div>
-      <div sx={sx.footerColumn}>
-        <div className='footer-parent' property='parentOrganization' typeof='CollegeOrUniversity' resource='#parentorg'>
-          <meta property='name' content='University of Notre Dame' />
-          <a href='https://www.nd.edu/' className='mark-footer' property='url logo' typeof='ImageObject' aria-label='University of Notre Dame'>
-            <img src={ndLogo} width='250' height='60' alt='University of Notre Dame' property='url' />
+        <div sx={sx.footerLeftColumn}>
+          <Link to='/'>
+            <img
+              src={marbleLogo}
+              sx={sx.image}
+              alt='MARBLE: Museums, Archives, Rare Books & Libraries Exploration'
+            />
+          </Link>
+          <a
+            href='https://sniteartmuseum.nd.edu/'
+            sx={sx.imageWrapper}
+          >
+            <img
+              src={sniteLogo}
+              sx={sx.image}
+              alt='Snite Art Museum'
+            />
+          </a>
+          <p className='contact-info'>
```

Comment:
I've been getting the following warning: 
```
index.js:2177 Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.
    in div (at Footer/index.js:37)
    in p (at Footer/index.js:36)
```
So can we get the `<p/>`s here changed to `<div/>` or <span/>`?

Reply:
Looks like you got the other instances of `<div/>` as a child of `<p/>`, get line 34 and line 41, then 👍 .